shiny/ui/_chat_tokenizer.py

```diff
@@ -54,7 +54,9 @@ def get_default_tokenizer() -> TokenizersTokenizer | None:
     except ImportError:
         warnings.warn(
             "`Chat` is unable obtain a default tokenizer without the `tokenizers` "
-            "package installed. Please `pip install tokenizers` or set "
+            "package installed. The tokenizer to use for calculating token counts, "
+            "which is required to impose `token_limits` in `.messages()`.\n"
+            "Please run `pip install tokenizers` or set "
             "`Chat(tokenizer=None)` to disable tokenization.",
```

Comment:
So when I see this message, my app is still working fine. So I'm guessing that it is falling back to None automatically? Would it be more accurate to say "set None to stop seeing this warning"?